askama/src/lib.rs

```diff
@@ -510,6 +510,21 @@ pub trait Template {
     }
     /// Renders the template to the given `writer` buffer
     fn render_into(&self, writer: &mut dyn std::fmt::Write) -> Result<()>;
+    /// Helper method which allocates a new `Vec<u8>` and renders into it
+    fn render_bytes(&self) -> Result<Vec<u8>> {
+        let mut buf = Vec::with_capacity(self.size_hint());
+        self.render_into_bytes(&mut buf)?;
+        Ok(buf)
+    }
+    /// Renders the template to the given `writer`. (It is recommended to use a buffered implementation.)
```

Comment:
The second sentence of the comment here should be on a separate line, separated from the first line by an empty line. On the first line, please remove the `.`.

Reply:
OK.